pkg/autoscaler/k8sclient/k8sclient.go

```diff
@@ -189,6 +189,13 @@ func findPatcher(kind string, groupVersions map[string]bool) (string, patchFunc,
 
 func findDeploymentPatcher(groupVersions map[string]bool) (string, patchFunc, error) {
 	// Find the best API to use - newest API first.
+	if groupVersions["apps/v1"] {
+		fn := func(client kubernetes.Interface, namespace, name string, pt types.PatchType, data []byte) error {
+			_, err := client.Apps().Deployments(namespace).Patch(name, pt, data)
```

Comment:
should be AppsV1(), right?

Reply:
Thanks! You are right, seems like we need to re-vendor client-go (https://github.com/kubernetes-incubator/cluster-proportional-vertical-autoscaler/blob/master/vendor/k8s.io/client-go/kubernetes/clientset.go#L48-L56) to include AppsV1().